ubuntu/16.04/usr/local/share/bootstrap/common_functions.sh

```diff
@@ -69,11 +69,6 @@ is_chown_forbidden() {
   return $?
 }
 
-is_vboxsf_mountpoint() {
-  grep -q "/app vboxsf" /proc/mounts
-  return $?
-}
-
```

Comment:
for instance osxfs has none of the chmod/chown issues as the other mount points, but still want to treat it as development mode by default. There are similar reasons for nfs, for the APP_USER_LOCAL isn't needed for it